### DISCUSSION TITLE
Can we get a pinch and zoom function on the video while streaming?

### DISCUSSION THREAD

**Discussion by jastsai306** [blakeblackshear/frigate#2118] [Ideas]
Is it possible to have this feature? It would be helpful for people with 4k resolutions to see up close. Thanks.

**Comment by mcdent** [comment]
I agree, this would be nice, on the home assistant app, I miss this feature.

**Reply by LaurenceGough** [reply]
On the HA App you can enable pinch to zoom in the app options which works great.

**Reply by arieltools** [reply]
fyi:
https://community.home-assistant.io/t/ha-web-interface-enable-pinch-to-zoom-on-mobile-devices/30389/18

**Comment by markfrancisonly** [comment]
and mouse down lasso select zoom with mouse scroll wheel  zoom in/out

**Reply by The-NinthWave** [reply]
@blakeblackshear 
Do you comment on those "would be nice" ?

Ability to use mouse on PC is something keeping people I know from moving to Frigate from Genius Vision and that I like to do myself on my phone when I look at my cameras with ONVIER.

**Comment by megakid** [comment]
Maybe this can be a Lovelace frigate card feature?

**Comment by nicoleise** [comment]
I miss this too, both on individual camera views and on multiple camera views like Birdseye. Precisely because of using 4k cameras to cover large areas and being used to be able to zoom in on the video to see stuff further away in detail.

Pinch to zoom, single touch to pan and double tap to zoom in/further in/out should both work on mobile devices if possible. I don't know how much of this is possible in a web page though.

**Comment by dominikjas** [comment]
I also would like to use the "normal" webinterface but because I can't zoom here I'm using the hass-card which is not as handy as the frigate interface in my opinion.
